components/Hero.js

```diff
@@ -28,28 +28,32 @@ export default function Hero({ className = ''}) {
             <strong>industry standard</strong> for defining asynchronous APIs.
           </Heading>
           <div className='flex flex-row items-center justify-center'>
-            <Button className="block md:inline-block" text="Read the docs" href="/docs" icon={<ArrowRight className="-mb-1 h-5 w-5" />} />
-            <SearchButton 
-              className="hidden sm:flex items-center text-left space-x-3 px-4 py-3 ml-2 bg-white border-secondary-500 border text-secondary-500 hover:text-white shadow-md bg-secondary-100 hover:bg-secondary-500 transition-all duration-500 ease-in-out rounded-md"
-            >
-              {({ actionKey }) => (
-                <>
-                  <IconLoupe />
-                  <span className="flex-auto">Quick search...</span>
-                  {actionKey && (
-                    <kbd className="font-sans font-semibold">
-                      <abbr
-                        title={actionKey.key}
-                        className="no-underline"
-                      >
-                        {actionKey.shortKey}
-                      </abbr>{' '}
-                      K
-                    </kbd>
-                  )}
-                </>
-              )}
-            </SearchButton>
+            <Button className="block md:inline-block" text="Read the docs" href="/docs" icon={<ArrowRight className="-mb-1 h-5 w-5" />} 
+            data-testid="Hero-Button"/>
+            {/* Wrap SearchButton with AlgoliaSearch component */}
+            <AlgoliaSearch>
```

Comment:
Specify the reason for wrapping the Algolia Search in this component.

Reply:
Here will writing the tests for Hero , Search Button was not able to access the context , and some other refs , thus wrapped AlgoliaSearch component around it